yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -39,6 +50,21 @@ export type JournalData = {
   currentStorageValue: Map<bigint, Map<bigint, Fr>>;
 };
 
+// TRANSITIONAL: This should be removed once the AVM is fully operational and the public executor is gone.
```

Comment:
I think the comment is not right. We will only be able to remove this once we have 1-kernel-per-enqueued-call, right? Same in other places.

Reply:
Changed comment (and changed same comment in the journal in a handful of places)